luigi/configuration/cfg_parser.py

```diff
@@ -38,8 +37,10 @@
 except ImportError:
     from configparser import ConfigParser, NoOptionError, NoSectionError
 
+from .base_parser import BaseParser
```

Comment:
I recall relative imports are discouraged, no?

Reply:
Why? We already have `from .tools import range` in project, and it's works. Relative imports little faster and better recognized some tools like isort.